.gitignore

```diff
@@ -2,4 +2,4 @@
 coverage
 node_modules
 npm-debug.log
-package-lock.json
+package-lock.json
```

Comment:
nit: For good git diff'ing, let's maybe ensure EOL at EOF's (in all introduced files)